### DISCUSSION TITLE
Transaction Simulation for an LUT

### DISCUSSION THREAD

**Discussion by jwalin12** [phantom/docs#255] [Transaction Simulation]
### dApp Name

Bridgesplit

### dApp URL

app.bridgesplit.com(gated)

### Program ID

multiple  programs

### Steps to reproduce

![Screen Shot 2024-06-26 at 6 04 17 PM](https://github.com/phantom/docs/assets/46630653/5b3aaf5b-231c-47d7-8f7b-8a22de6917ac)

Hello,

I have a series of 3 transactions, the first of which creates a LookupTable. The following transactions use that lookup table. Currently, I have the users sign all 3 transactions at once. 

My BE is sending the transactions. I await for confirmed on the first transaction, so that I know the LUT is created. Once that is done,  I send the other 2 transactions. The transactions all work, but the simulation in phantom fails because I think the versioned transactions that depend on the LUT in transaction 1 and cannot simulate it because the LUT is not on-chain yet.  

In similar cases on our app when transactions depend on each other phantom is able to simulate them correctly, so I think it is an issue with LUTs specifically

It's very important for us to only have the user sign once for UX reasons, however since the phantom simulation is failing the transactions look suspect. Let me know how I can get around this without requiring multiple sign and sends if possible. 

Thanks,
Jwalin

**Comment by adamdelphantom** [comment]
@jwalin12 Apologies for the delay in response here. Is this issue still persistent?
